### PR TITLE
add gemviz package

### PR DESCRIPTION
**Pull request by prjemian**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by prjemian** [comment]
Ready for review.

**Comment by rodolakis** [comment]
I am willing to be a co-maintainer for this package.

**Comment by prjemian** [comment]
@conda-forge/help-python all test pass. This PR is ready for review.

File: recipes/gemviz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+{% set version = "0.1.0" %}
+{% set sha256 = "740b4be1938320efd84ab6845aadaeb71c3047de1c927625613539e30b6e844a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (ocefpaf) [inline recipes/gemviz/meta.yaml:11]: No need to template something that will be used only once.

```suggestion

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: 740b4be1938320efd84ab6845aadaeb71c3047de1c927625613539e30b6e844a
```

Comment (prjemian) [reply recipes/gemviz/meta.yaml:11]: _au contraire_, this definition has value to the maintainer.  It places the version and sha256 adjacent to each other, reducing the chance of inconsistency errors experienced in previous submissions.  Is removal absolutely necessary?

Comment (ocefpaf) [reply recipes/gemviz/meta.yaml:11]: We have a bot that updates the recipe. We don't update that manually ever.

Comment (prjemian) [reply recipes/gemviz/meta.yaml:11]: Most often in the recipes I maintain, they are update be me, not by bot.  The bot misses some development details.

**Comment by prjemian** [comment]
@ocefpaf - Your changes have been accepted.  Thanks for the feedback.  Ready for review.

**Comment by prjemian** [comment]
@ocefpaf, Thanks!

**Pull request merged**